src/analysis/config.d

```diff
@@ -140,8 +140,11 @@ struct StaticAnalysisConfig
 	string alias_syntax_check = Check.disabled;
 
 	@INI("Checks for else if that should be else static if")
-    string static_if_else_check = Check.disabled;
+	string static_if_else_check = Check.disabled;
```

Comment:
No, actually dscanner sources mostly use tab indentation. Here I've just replaced 4 spaces by a tab. In a diff GH displays 8 spaces for a tab instead of 4 (otherwise the diff could look like a "no diff") so it gives the feeling that there's an issue. You can [check the whole file](https://github.com/BBasile/Dscanner/blob/8f4ab6671a5bca8908d6a278bb77b89ea6075fb5/src/analysis/config.d).